terminal_location.go

```diff
@@ -15,13 +21,13 @@ type TerminalLocationListParams struct {
 // TerminalLocation is the resource representing a Stripe terminal location.
 type TerminalLocation struct {
 	APIResource
-	Address     *AccountAddressParams `json:"address"`
-	Deleted     bool                  `json:"deleted"`
-	DisplayName string                `json:"display_name"`
-	ID          string                `json:"id"`
-	Livemode    bool                  `json:"livemode"`
-	Metadata    map[string]string     `json:"metadata"`
-	Object      string                `json:"object"`
+	Address     *AccountAddress   `json:"address"`
```

Comment:
Is that one a breaking change?

Reply:
It's either a breaking change or a bugfix.

AccountAddressParams right now only has `form` annotatations and not `json` annotations, so I think right now it's pretty broken and these will always be zero values.